src/modules/account/hooks/useCurrentAccount.js

```diff
@@ -65,7 +71,10 @@ export function useCurrentAccount() {
         }
       }
     }
+    // Remove toast between account switches
+    toast.dismiss();
```

Comment:
This is the only real change in this file? Others are formatting? Just want to know because i will get merge conflict on this file if this PR is merged before my PR

Reply:
Yeah, they're formatting changes